### ISSUE TITLE
[Question][Jira Plugin] How to handle "429 - Too many requests" issue from atlassian cloud

### ISSUE THREAD

**Issue by lshmouse** [apache/incubator-devlake#2932]
## Question

After add a blueprint for a project from jira, the jira plugin always run failed for: 429 - Too many requests.

For there is no way to set the rate_limit_per_hour in config-ui, i update it in mysql by the command 
```
 update _tool_jira_connections set rate_limit_per_hour = 1800 where id =1;
```
But we encountered the same issue somtimes.

Any suggestion are welcomed, thanks~

## Screenshots
If applicable, add screenshots to help explain.

## Additional context
```
time="2022-09-02 10:13:02" level=warning msg=" [task #16] [jira] retry #1 for http code error[429]:[<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><title>Oops - an error has occurred</title><link type='text/css' rel='stylesheet' href='/static-assets/metal-all.css' media='all'><script src='/static-assets/metal-all.js'></script><meta name="decorator" content="none" /></head><body class=" error-page error429"><script type="text/javascript">document.body.className += " js-enabled";</script><div id="page"><header id="header" role="banner"></header><!-- #header --><section id="content" role="main"><div class="aui-page-panel"><div class="aui-page-panel-inner"><section class="aui-page-panel-content lowerContent"><div id="error-state"><span class="error-type"></span><h1>Something went wrong</h1><p>Try reloading the page, then check our <a href="https://status.atlassian.com">Statuspage</a> for any current outages. If there are no relevant outages, create a <a href="https://support.atlassian.com/contact/">support request</a> so we can help you out.</p><p>If you create a request, include the following so we can help you as fast as possible:</p><ul><li>Error type: <strong>429 - Too many requests</strong></li><li>Log reference: <strong>0d4104e5-ad74-4eca-b624-cc46a73bd1ef</strong></li></ul></div></section><!-- .aui-page-panel-content --></div><!-- .aui-page-panel-inner --></div><!-- .aui-page-panel --></section><!-- #content --><footer id="footer" role="contentinfo"><section class="footer-body"><div id="footer-logo"><a href="http://www.atlassian.com/" rel="nofollow">Atlassian</a></div></section></footer><!-- #footer --></div><!-- #page --></body></html>]"
time="2022-09-02 10:13:02" level=info msg=" [task #16] [jira] [collectIssues] end api collection error: %!w(*errors.errorString=&{[http code error[429]:[<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><title>Oops - an error has occurred</title><link type='text/css' rel='stylesheet' href='/static-assets/metal-all.css' media='all'><script src='/static-assets/metal-all.js'></script><meta name="decorator" content="none" /></head><body class=" error-page error429"><script type="text/javascript">document.body.className += " js-enabled";</script><div id="page"><header id="header" role="banner"></header><!-- #header --><section id="content" role="main"><div class="aui-page-panel"><div class="aui-page-panel-inner"><section class="aui-page-panel-content lowerContent"><div id="error-state"><span class="error-type"></span><h1>Something went wrong</h1><p>Try reloading the page, then check our <a href="https://status.atlassian.com">Statuspage</a> for any current outages. If there are no relevant outages, create a <a href="https://support.atlassian.com/contact/">support request</a> so we can help you out.</p><p>If you create a request, include the following so we can help you as fast as possible:</p><ul><li>Error type: <strong>429 - Too many requests</strong></li><li>Log reference: <strong>5d044ad0-1188-45a3-aa21-88af00f82ec0</strong></li></ul></div></section><!-- .aui-page-panel-content --></div><!-- .aui-page-panel-inner --></div><!-- .aui-page-panel --></section><!-- #content --><footer id="footer" role="contentinfo"><section class="footer-body"><div id="footer-logo"><a href="http://www.atlassian.com/" rel="nofollow">Atlassian</a></div></section></footer><!-- #footer --></div><!-- #page --></body></html>]]})"
```

**Comment by abeizn** [comment]
@lshmouse 
We have implemented this feature in v0.13.0 version. 
https://github.com/apache/incubator-devlake/tree/release-v0.13 , it is in the process of releasing.
![image](https://user-images.githubusercontent.com/101256042/188058002-acf5f9a8-a221-4d28-8b78-b9ba5e1b065d.png)

**Comment by lshmouse** [comment]
> @lshmouse We have implemented this feature in v0.13.0 version. https://github.com/apache/incubator-devlake/tree/release-v0.13 , it is in the process of releasing. ![image](https://user-images.githubusercontent.com/101256042/188058002-acf5f9a8-a221-4d28-8b78-b9ba5e1b065d.png)

Perfect~ When will the 0.13 be release? Maybe I can build docker image using latest code.

**Comment by abeizn** [comment]
> 
@lshmouse yes, now the 0.13 version is release.
